TestingTravis/TestClass2.swift

```diff
@@ -12,5 +12,6 @@ class TestClass2{
     
     init(){
```

Comment:
<table data-meta="generated_by_1">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/e663d8fc47ea5cca509e24d98b40c79599d1e11e/TestingTravis/TestClass2.swift#L13"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>TestClass2.swift:13</code></td>
    </tr>
  </tbody>
</table>